### ISSUE TITLE
ZapZap Settings don't persit

### ISSUE THREAD

**Issue by Novantric** [rafatosta/zapzap#219]
Hello,
I just installed ZapZap and so far I really like it, however there's one major bug disrupting my experience.
Whenever I close the app (click the system tray icon > quit) and relaunch all the app settings are reverted to default, yet I remain signed into my one profile.

My customisation are simply:
- Remove sidebar for one account
- Light theme systray icon
- Hide donation message
- Start minimisedon system boot

I installed ZapZap from Flathub via Gnome software on Ubuntu 24.04 LTS.
Thanks! I can provide additional info if needed.

**Comment by rafatosta** [comment]
> Hello, I just installed ZapZap and so far I really like it, however there's one major bug disrupting my experience. Whenever I close the app (click the system tray icon > quit) and relaunch all the app settings are reverted to default, yet I remain signed into my one profile.
> 
> My customisation are simply:
> 
> * Remove sidebar for one account
> * Light theme systray icon
> * Hide donation message
> * Start minimisedon system boot
> 
> I installed ZapZap from Flathub via Gnome software on Ubuntu 24.04 LTS. Thanks! I can provide additional info if needed.

I have no idea what it is. But, I'm getting a lot of weird bugs when using Flatpak on Ubuntu.

**Comment by Novantric** [comment]
That must be annoying. Keep us posted.

**Comment by rafatosta** [comment]
> That must be annoying. Keep us posted.

My suspicions are that Ubuntu wants to make it harder to use Flatpak and force users to Snap. Which wouldn't be anything new!

**Comment by Novantric** [comment]
Weird though, I've only had this issue once before with Prism Launcher, which I solved by installing the .deb.
If that's simple enough you could do the same?

**Comment by WolfganP** [comment]
I use MXlinux (debian based) and no issues whatsoever installing the flatpak and using zapzap

**Comment by Novantric** [comment]
I found [this](https://ubuntuforums.org/showthread.php?t=2499125) page on the Ubuntu forums recently with people having the same issue, but on Kubuntu. How annoying. Weird how some Flatpaks run just fine though.

**Comment by rafatosta** [comment]
> I found [this](https://ubuntuforums.org/showthread.php?t=2499125) page on the Ubuntu forums recently with people having the same issue, but on Kubuntu. How annoying. Weird how some Flatpaks run just fine though.
> 
> Maybe we could adjust something in Flatseal? 

Honestly, nowadays Ubuntu is no longer worth it. Even distros like Debian and PopOS deliver a better experience than Ubuntu. 

After Snap, Ubuntu is no longer the same. My recommendation is to use another distro!

**Comment by Novantric** [comment]
That's fair, but that's your view and not necessarily applicable for everyone else.
I for one don't wanna cause it took me hours to setup my installation to where I'm happy with it.
If you can't do anything to fix this issue then close it, the Flatpak devs state this should be brought up with the devs, Like I have done.
https://github.com/flatpak/flatpak/issues/5869#issuecomment-2225480509

**Comment by rafatosta** [comment]
> That's fair, but that's your view and not necessarily applicable for everyone else.
> I for one don't wanna cause it took me hours to setup my installation to where I'm happy with it.
> If you can't do anything to fix this issue then close it, the Flatpak devs state this should be brought up with the devs, Like I have done.
> https://github.com/flatpak/flatpak/issues/5869#issuecomment-2225480509

My answer is the same as the Flatpak dev's: it's not a ZapZap bug.

I understand your user point of view, However, I can't do anything because there are no errors in the app. But rather how Ubuntu handles Flatpaks.

**Comment by flokosx** [comment]
I think this problem is recurrent in several flatpaks.
Thanks to colleagues on forums, I managed to solve my other problem. I don't understand much about the technicalities of the thing but it's working as expected now.

in short, for a beginner like me, I installed:
sudo apt install apparmor-utils

them:
sudo aa-disable /usr/bin/bwrap

Source: https://bugs.launchpad.net/ubuntu/+source/apparmor/+bug/2072811
